tests/e2e/performance/csi_tests/test_pvc_multi_clone_performance.py

```diff
@@ -3,37 +3,85 @@
 import os
 import tempfile
 import time
+from uuid import uuid4
+from ocs_ci.framework import config
+import statistics
 
 import yaml
 import pytest
 
+from ocs_ci.ocs.perftests import PASTest
+from ocs_ci.ocs.perfresult import ResultsAnalyse
 from ocs_ci.framework.testlib import (
     skipif_ocs_version,
     skipif_ocp_version,
-    E2ETest,
     performance,
 )
+from ocs_ci.helpers.helpers import get_full_test_logs_path
 from ocs_ci.ocs import constants, exceptions
 from ocs_ci.ocs.cluster import CephCluster
 from ocs_ci.utility.utils import ocsci_log_path
 from ocs_ci.helpers import performance_lib
 
 log = logging.getLogger(__name__)
 
-SKIP_REASON = "Skipping multiple clone tests until github issue 5198 is addressed"
+# SKIP_REASON = "Skipping multiple clone tests until github issue 5198 is addressed"
```

Comment:
are we removing this skip?

Reply:
github https://github.com/red-hat-storage/ocs-ci/issues/5198 issues is due to teardown in test_multi_snapshot.py tests, don't think we are observing this problem in recent runs, correct me if I am wrong here.

Reply:
+1 for @ramkiperiy , this line should be deleted and not markout

Reply:
yes, I've executed the test locally and it does not leave leftovers. I think the skip should be removed, and if we see any problem in the future - we'll address it then. There is not reason as I see it to skip the test now.

Reply:
@Avilir sure, will delete it. Thanks!